deploy/examples/v2/custom_images_registry.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: image-registry
+spec:
+  environment: rhpam-trial
+  imageRegistry:
+    insecure: true
+    registry: registry.example.com:5000
+  objects:
+    console:
+      image: console-image
+      imageTag: "1.2"
+    servers:
+      - image: server-image
+        imageTag: "1.2"
+      - image: server-image2
+        imageTag: "1.1"
```

Comment:
What prompted this change? I would not worry about trying to support two different versions of KIE server in the same deployment, if that's the motivation. I see that it adds the flexibility to use custom image names; is that the reason?